chili-epfl/FROG
# Sign Up modal

This PR adds a modal to allow users to create a verified account in FROG. For now this modal can be pulled up using the Sign up/login button on the single activity page. The following validation has also been added to the fields:
- Email: Has to be a valid email in the format: xyz@domain.com
- Password: Has to be minimum 5 characters
- Display Name: Has to be minimum 1 character

**Testing instructions**
- Open the modal and enter your information 
- On clicking submit and if the information is valid, an alert should pop up with a success message and it should redirect to the main page. Check the database for the new account you created. 
- Try to create an account with invalid inputs/ with an email already used and you should get an error message


**Asana issue**
https://app.asana.com/0/1129712605359574/1129712605359576/f
